src/azul/indexer/document.py

```diff
@@ -558,7 +566,10 @@ def field_types(cls, field_types: FieldTypes) -> FieldTypes:
         return {
             **super().field_types(field_types),
             'num_contributions': pass_thru_int,
-            'bundles': pass_thru_jsons
+            'bundles': {
+                'uuid': pass_thru_str,
+                'version': pass_thru_str,
+            }
```

Comment:
Why are you touching this? And if it is an independent change, why is it not a separate commit?

Reply:
Previously the `FieldType` of `bundles.version` was not needed due to it never being `None`, not being a facet, never used as a filter, and not being an aggregated field.
Now however the `FieldType` of `bundles.version` is required because `bundles.version` is a field that can be sorted by, so the `FieldType` of the field is needed to get its `es_sort_mode`.